#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_mysql_secure_installation.dpatch by Norbert Tretkowski <nobse@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Don't fail on passwords which needs quoting

@DPATCH@
--- mysql-dfsg-5.0-5.0.51a.orig/scripts/mysql_secure_installation.sh	2008-01-11 15:43:40.000000000 +0100
+++ mysql-dfsg-5.0-5.0.51a/scripts/mysql_secure_installation.sh	2009-01-24 22:58:45.000000000 +0100
@@ -47,7 +47,7 @@
     echo "# mysql_secure_installation config file" >$config
     echo "[mysql]" >>$config
     echo "user=root" >>$config
-    echo "password=$rootpass" >>$config
+    echo "password='$rootpass'" >>$config
 }
 
 get_root_password() {
